service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -95,13 +94,7 @@ public Other extractConfidentialDetails() {
 
     @Override
     public Other addConfidentialDetails(Party party) {
-        Telephone telephoneNumber = ofNullable(party.getTelephoneNumber()).orElse(Telephone.builder().build());
-
-        return this.toBuilder()
-            .name(party.getFirstName())
-            .address(party.getAddress())
-            .telephone(ofNullable(telephoneNumber.getTelephoneNumber()).orElse(""))
-            .build();
+        return null;
```

Comment:
This method shouldn't return null? We could update this method to the new way we are using it here perhaps, this one..

    private Other addConfidentialDetails(Other confidentialOther, Element<Other> other) {
        return other.getValue().toBuilder()
            .telephone(confidentialOther.getTelephone())
            .address(confidentialOther.getAddress())
            .build();
    }

Reply:
we can't update method because addConfidentialDetails is implementation of interface. I either have to add new method to interface and implement dummy in Child and Respondent or do it this way..

Reply:
OK sure then maybe this is only option :)